content/en/installing-pixie/03-install-guides/01-community-cloud-for-pixie.md

```diff
@@ -0,0 +1,120 @@
+---
+title: "Community Cloud for Pixie"
+metaTitle: "Install | Install Guides | Community Cloud for Pixie"
+metaDescription: "Getting started guide to setup Pixie"
+order: 1
+---
+
+Please review Pixie's [requirements](/installing-pixie/requirements) to make sure that your Kubernetes cluster is supported. 
+
+## 1. Sign up
+
+Visit our [product page](https://work.withpixie.ai/) and sign up with your google or gsuite account.
+
+## 2. Set up a Kubernetes environment (optional)
+
+If you don't have a Kubernetes cluster available, you can set up Minikube as a local sandbox environment. Instructions are available [here](/installing-pixie/setting-up-k8s/minikube-setup).
+
+## 3. Install the Pixie CLI
```

Comment:
To shorten this doc, I would replace step 3 with the following:

The easiest way to install Pixie's CLI is using the install script:

``` bash
# Copy and run command to install the Pixie CLI.
bash -c "$(curl -fsSL https://withpixie.ai/install.sh)"
```

The CLI can also be installed by directly downloading the binary or with Docker or the Debian package. For directions, see the [CLI install docs](/installing-pixie/install-schemes/cli/) .